rosdep/base.yaml

```diff
@@ -3644,14 +3644,32 @@ wx-common:
 wxpython:
   arch: [wxpython]
   centos: [wxPython-devel]
-  debian: [python-wxgtk2.8]
+  debian:
+    jessie: [python-wxgtk3.0]
+    sid: [python-wxgtk2.8]
```

Comment:
Sid should be 3.0 it's the experimental rolling release for debian. https://packages.debian.org/sid/python-wxgtk3.0